app.d/prometheus.py

```diff
@@ -146,6 +149,18 @@ def update(app: ApplicationState):
     result_dynamic_average = result_dynamic.dropColumns("DateTime", "Job", "Instance").avgBy("PrometheusQuery")
     app.setField("result_dynamic_average", result_dynamic_average)
 
+    #Downsampling examples
+    result_dynamic_downsampled_average = result_dynamic.update("DateTimeMinute = lowerBin(DateTime, 60000000000)")\
+        .dropColumns("DateTime")\
+        .avgBy("PrometheusQuery", "DateTimeMinute", "Job", "Instance")
+    app.setField("result_dynamic_downsampled_average", result_dynamic_downsampled_average)
+
+    result_dynamic_downsampled_tail = result_dynamic.tail(20)
+    app.setField("result_dynamic_downsampled_tail", result_dynamic_downsampled_tail)
+
+    result_dynamic_ema = result_dynamic.view("PrometheusQuery", "EMA = prometheus_metrics_ema.update(DateTime, Value, PrometheusQuery, Job, Instance)").lastBy("PrometheusQuery").tail(10)
```

Comment:
Break this into multiple lines for readability.

Reply:
Done 👍